src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure implements Area, ColorCreator {
```

Comment:
make your Figure implement Area and you won't need to do it in each class

Reply:
Done. =)